tap_zuora/apis.py

```diff
@@ -154,7 +154,7 @@ def get_payload(state, stream, partner_id):
 
     @staticmethod
     def get_url(client):
-        return APIS[(client.european, client.sandbox)]
+        return APIS[(client.sandbox, client.european)]
```

Comment:
When the `APIS` set of tuples is `(PROD / SANDBOX, NA / EU)` you should access it correctly.